app/client/ui/WebhookPage.ts

```diff
@@ -37,33 +37,39 @@ const WEBHOOK_COLUMNS = [
     id: 'vt_webhook_fc1',
     colId: 'tableId',
     type: 'Choice',
-    label: 'Table',
+    label: t('Table'),
     // widgetOptions are configured later, since the choices depend
     // on the user tables in the document.
   },
   {
     id: 'vt_webhook_fc2',
     colId: 'url',
     type: 'Text',
-    label: 'URL',
+    label: t('URL'),
   },
   {
     id: 'vt_webhook_fc3',
     colId: 'eventTypes',
     type: 'ChoiceList',
-    label: 'Event Types',
+    label: t('Event Types'),
     widgetOptions: JSON.stringify({
       widget: 'TextBox',
       alignment: 'left',
       choices: ['add', 'update'],
       choiceOptions: {},
     }),
   },
+  {
+    id: 'vt_webhook_fc10',
+    colId: 'columnIds',
+    type: 'Text',
+    label: t('Columns to check when update (separated by ;)'),
```

Comment:
Maybe `Filter for changes in these columns (semicolon-separated ids)`

Might be too long?